core/server/master/src/test/java/alluxio/master/AlluxioMasterProcessTest.java

```diff
@@ -156,7 +158,7 @@ private void waitForServing(ServiceType service) throws TimeoutException, Interr
       } catch (IOException e) {
         throw new RuntimeException(e);
       }
-    });
+    }, WaitForOptions.defaults().setTimeoutMs(5 * Constants.MINUTE_MS));
```

Comment:
what was this change for?

Reply:
Previously, this test would hang for multiple hours if the master/worker failed to come up. Now, it will time out.

Reply:
But, is this unrelated to jvm pause monitor tests? If so, I think this should be its own PR/commit.

Reply:
This was kind-of related. A bug in the monitor usage caused this test to fail (hang) for 2+ hours. It was very hard to track down with jenkins because I couldn't tell which tests are running when jenkins aborts the job. It would have been easier to find had this test contained a timeout.